db/project/projectOrchestrator.js

```diff
@@ -0,0 +1,121 @@
+const { Project } = require("./project");
+
+module.exports.ProjectOrchestrator = class {
+   constructor(jsonSource) {
+      this._nextAvailableIndex = 0;
+      this._projects = [];
```

Comment:
see this line, `jsonSource` is a buffer, but it's empty. So convert the `Buffer` to a string first before checking.

Reply:
Thank you! I was too tired yesterday 😂